{"uuid": "c2641d00-d3db-4609-8941-419c3e06c2fd", "children": ["d8670b9e-e6e9-4f2c-a96e-819badcf38ca"], "befores": [{"name": "login_fixture", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"D:\\anaconda\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\anaconda\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\anaconda\\lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\ApiTest\\test_case\\conftest.py\", line 31, in login_fixture\n    result= loginApi.login(username,password,code)\n  File \"D:\\ApiTest\\api\\login_ok.py\", line 45, in login\n    print(r)\n  File \"D:\\ApiTest\\api\\login_ok.py\", line 45, in login\n    print(r)\n  File \"_pydevd_bundle\\pydevd_cython_win32_39_64.pyx\", line 1589, in _pydevd_bundle.pydevd_cython_win32_39_64.ThreadTracer.__call__\n  File \"_pydevd_bundle\\pydevd_cython_win32_39_64.pyx\", line 929, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.trace_dispatch\n  File \"_pydevd_bundle\\pydevd_cython_win32_39_64.pyx\", line 920, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.trace_dispatch\n  File \"_pydevd_bundle\\pydevd_cython_win32_39_64.pyx\", line 317, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.do_wait_suspend\n  File \"D:\\pycharm\\PyCharm Community Edition 2022.1\\plugins\\python-ce\\helpers\\pydev\\pydevd.py\", line 1155, in do_wait_suspend\n    self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread)\n  File \"D:\\pycharm\\PyCharm Community Edition 2022.1\\plugins\\python-ce\\helpers\\pydev\\pydevd.py\", line 1170, in _do_wait_suspend\n    time.sleep(0.01)\n"}, "start": 1723729321776, "stop": 1723729335971}], "start": 1723729321775, "stop": 1723729336883}